coinstacks/common/api/src/models.ts

```diff
@@ -33,6 +33,7 @@ export interface InternalServerError {
 export interface Account {
   balance: string
   pubkey: string
+  path?: string
```

Comment:
Add path to `BitcoinAccount`

Reply:
Still want to pull this out and extend `BitcoinAccount`

Reply:
Ok my bad I see what you're sayin.

Reply:
Looks like it was added, so we can just remove